types/stripe-js/payment-request.d.ts

```diff
@@ -14,6 +14,13 @@ export interface PaymentRequest {
    */
   show(): void;
 
+  /**
+   * Closes the browser’s payment interface.
+   *
+   * WARNING: NOT OFFICIALLY SUPPORTED BY STRIPE
+   */
+  abort?: () => void;
```

Comment:
I've confirmed we expose an `abort` function in both `StripePaymentRequest` and `ApplePaySession`.

```suggestion
  /**
   * Closes the browser’s payment interface.
   */
  abort: () => void;
```